src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/legalrepresentative/LegalRepresentativeRemissionDecisionPartiallyApprovedPersonalisation.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.legalrepresentative;
+
+import static java.util.Objects.requireNonNull;
+import static uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition.*;
+import static uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition.LEGAL_REPRESENTATIVE_EMAIL_ADDRESS;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.EmailNotificationPersonalisation;
+
+@Component
+public class LegalRepresentativeRemissionDecisionPartiallyApprovedPersonalisation implements EmailNotificationPersonalisation {
+
+    private final String partiallyApprovedTemplateId;
+    private final String iaExUiFrontendUrl;
+
+    public LegalRepresentativeRemissionDecisionPartiallyApprovedPersonalisation(
+        @Value("${govnotify.template.recordRemissionDecision.partiallyApproved.legalRep.email}") String partiallyApprovedTemplateId,
+        @Value("${iaExUiFrontendUrl}") String iaExUiFrontendUrl
+
+    ) {
+        this.partiallyApprovedTemplateId = partiallyApprovedTemplateId;
+        this.iaExUiFrontendUrl = iaExUiFrontendUrl;
+    }
+
+    @Override
+    public String getTemplateId(AsylumCase asylumCase) {
+        return partiallyApprovedTemplateId;
+    }
+
+    @Override
+    public Set<String> getRecipientsList(AsylumCase asylumCase) {
+        return Collections.singleton(asylumCase
+            .read(LEGAL_REPRESENTATIVE_EMAIL_ADDRESS, String.class)
+            .orElseThrow(() -> new IllegalStateException("legalRepresentativeEmailAddress is not present")));
+    }
+
+    @Override
+    public String getReferenceId(Long caseId) {
+        return caseId + "_REMISSION_DECISION_PARTIALLY_APPROVED_LEGAL_REPRESENTATIVE";
+    }
+
+    @Override
+    public Map<String, String> getPersonalisation(AsylumCase asylumCase) {
+        requireNonNull(asylumCase, "asylumCase must not be null");
+
+        return ImmutableMap
+            .<String, String>builder()
+            .put("appealReferenceNumber", asylumCase.read(APPEAL_REFERENCE_NUMBER, String.class).orElse(""))
```

Comment:
Similar to the other personalisation classes for legal rep, you will need to add the following to the map `.putAll(customerServicesProvider.getCustomerServicesPersonalisation())`

Reply:
Added the telephoneNumber and emailAddress. Modified the template in Gov Notify